sfge/src/main/java/com/salesforce/graph/ops/expander/ApexPathExpander.java

```diff
@@ -433,6 +433,9 @@ void initializeClassStaticScope(String className)
     void visit(ApexPath path)
             throws PathExcludedException, MethodPathForkedException, RecursionDetectedException,
                     ReturnValueInvalidCollapsedException, PathCollapsedException {
+        for (PathExpansionObserver observer : config.getExpansionObservers()) {
+            observer.onPathVisit(path);
+        }
```

Comment:
Could you extract this to a new method and invoke it here so that this is easy to read in the flow? I'm planning to do this for a few more things in this class as well to increase readability.